cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -74,6 +75,55 @@ type wizard struct {
 	in         *bufio.Reader // Wrapper around stdin to allow reading user input
 }
 
+type wizardOpt struct {
+	desc          string
+	invoke        func()
+	rinkeby       bool
+	transcoder    bool
+	notTranscoder bool
+}
+
+func (w *wizard) initializeOptions() []wizardOpt {
+	options := []wizardOpt{
+		{desc: "Get node status", invoke: func() { w.stats(w.transcoder) }},
+		{desc: "View protocol parameters", invoke: w.protocolStats},
+		{desc: "List registered transcoders", invoke: func() { w.registeredTranscoderStats() }},
```

Comment:
Why do we need a `func()` here instead of just using `w.registeredTranscoderStatus()`?

Reply:
`registeredTranscoderStats` has this signature:
`func (w *wizard) registeredTranscoderStats() map[int]common.Address`
it doesn't fit into `invoke` definition (because it has return value)